slides/2-html-css.html

```diff
@@ -714,6 +755,21 @@ <h2><i class="fas fa-paint-brush"></i> Selectors</h2>
                         https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Selectors
                     </textarea>
                 </section>
+                <section data-markdown style="font-size: 0.8em;">
```

Comment:
New content : chaining selectors (will be necessary later on, so need a slide for that)